src/lib/config.js

```diff
@@ -195,12 +194,14 @@ function getLocalConfig () {
 
   const expiry = {}
   expiry.minMessageWindow =
-    Config.getEnv(envPrefix, 'MIN_MESSAGE_WINDOW') || 1 // seconds
-  expiry.maxHoldTime = Config.getEnv(envPrefix, 'MAX_HOLD_TIME') || 10 // seconds
+    +Config.getEnv(envPrefix, 'MIN_MESSAGE_WINDOW') || 1 // seconds
+  expiry.maxHoldTime = +Config.getEnv(envPrefix, 'MAX_HOLD_TIME') || 10 // seconds
 
   // The spread is added to every quoted rate
   const fxSpread = Number(Config.getEnv(envPrefix, 'FX_SPREAD')) || 0.002 // = .2%
 
+  const slippage = +Config.getEnv(envPrefix, 'SLIPPAGE') || 0.001 // = 0.1%
```

Comment:
Could you put these hard coded defaults up at the top of the file as `const DEFAULT_MIN_MESSAGE_WINDOW` or something like that?